packages/upgrade/src/transforms/hooksToOptions.ts

```diff
@@ -266,7 +266,28 @@ export default function transform(
                     memberExpr,
                   );
                 }
-                memberExpr.property = j.identifier('queryFilter');
+
+                /**
+                 * If no input, use pathFilter
+                 * If input has cursor, use infiniteQueryFilter
+                 * Otherwise, use queryFilter
+                 */
+                if (!callExpr.arguments.length) {
```

Comment:
What if the second argument is passed? Can we check if the first arg is an undefined literal?

Reply:
when would that be the case in the old API?

Reply:
could you do `utils.post.invalidate(undefined, { predicate: ... })` before for example?

Reply:
Yes exactly, that is a valid usage